components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -85,59 +85,13 @@ export default function PrebuildLogs(props: PrebuildLogsProps) {
             props.onInstanceUpdate(workspaceInstance);
         }
         switch (workspaceInstance?.status.phase) {
-            // unknown indicates an issue within the system in that it cannot determine the actual phase of
-            // a workspace. This phase is usually accompanied by an error.
-            case "unknown":
-                break;
-
             // Preparing means that we haven't actually started the workspace instance just yet, but rather
             // are still preparing for launch. This means we're building the Docker image for the workspace.
             case "preparing":
-                getGitpodService().server.watchWorkspaceImageBuildLogs(workspace!.id);
-                break;
-
-            // Pending means the workspace does not yet consume resources in the cluster, but rather is looking for
-            // some space within the cluster. If for example the cluster needs to scale up to accomodate the
-            // workspace, the workspace will be in Pending state until that happened.
-            case "pending":
-                break;
-
-            // Creating means the workspace is currently being created. That includes downloading the images required
-            // to run the workspace over the network. The time spent in this phase varies widely and depends on the current
-            // network speed, image size and cache states.
-            case "creating":
-                break;
-
-            // Initializing is the phase in which the workspace is executing the appropriate workspace initializer (e.g. Git
-            // clone or backup download). After this phase one can expect the workspace to either be Running or Failed.
-            case "initializing":
-                break;
-
-            // Running means the workspace is able to actively perform work, either by serving a user through Theia,
-            // or as a headless workspace.
-            case "running":
-                break;
-
-            // Interrupted is an exceptional state where the container should be running but is temporarily unavailable.
-            // When in this state, we expect it to become running or stopping anytime soon.
-            case "interrupted":
-                break;
-
-            // Stopping means that the workspace is currently shutting down. It could go to stopped every moment.
-            case "stopping":
```

Comment:
I'm curious to know your thoughts on removing all these "empty" cases. I get that they currently have no effect here, but we also have a history of "forgetting" to handle specific instance phases which are important to handle, so I initially felt like listing them all here, with appropriate comments, would make it easier not to forget about them in future refactors. 💭

Reply:
The reasoning:
* We have types that allow us to discover the cases
* They are documented against the underlying `gitpod-protocol` already, this is just duplication here.